### PR TITLE
docs: add synchronous replication

### PR DESCRIPTION
**Pull request by BusyJay**
### What is changed and how it works?

Add instructions to configure replication mode.

### Check List 

Tests 

- No code

### Release note 

- No release note

### CODE REVIEW

File: docs/how-to/configure/synchronous-replication.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Note this is still an experimental feature.
+
+## How to enable synchronous replication
+
+Replication mode is controlled by PD. You can use following example configuration to setup a cluster with synchronous replication from scratch:
+
+```toml
+[replication-mode]
+replication-mode = "dr_auto_sync"
```
Comment (disksing) [inline docs/how-to/configure/synchronous-replication.md:21]: latest version change all `dr_auto_sync` to `dr-auto-sync`.

File: docs/how-to/configure/synchronous-replication.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+title: Synchronous Replication
+summary: Learn how to configure synchronous replication in dual data centers.
+category: how-to
+---
+
+# Synchronous Replication
+
+This section describes how to configure synchronous replication in dual data centers.
+
+One of the data centers is primary, and the other is dr. When a region has odd number of replicas, primary will have more. When dr is down for more than configured time, replication will be changed to be asynchronous and primary will serve requests by its own. Asynchronous replication is used by default.
```
Comment (NingLin-P) [inline docs/how-to/configure/synchronous-replication.md:11]: Maybe we should describe more detail about these two replication modes (like how a raft log is considered committed).

Comment (NingLin-P) [inline docs/how-to/configure/synchronous-replication.md:11]:
```suggestion
One of the data centers is primary, and the other is dr. When a region has odd number of replicas, primary will be placed more replicas. When dr is down for more than configured time, replication will be changed to be asynchronous and primary will serve requests by its own. Asynchronous replication is used by default.
```

File: docs/how-to/configure/synchronous-replication.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Note this is still an experimental feature.
+
+## How to enable synchronous replication
+
+Replication mode is controlled by PD. You can use following example configuration to setup a cluster with synchronous replication from scratch:
```
Comment (NingLin-P) [inline docs/how-to/configure/synchronous-replication.md:17]:
```suggestion
Replication mode is controlled by PD. You can use following example PD configuration to setup a cluster with synchronous replication from scratch:
```

File: docs/how-to/configure/synchronous-replication.md (hunk 2 above)
Comment (BusyJay) [reply docs/how-to/configure/synchronous-replication.md:11]: It's a configuration documents, just talk how to enable the feature. A details introduction is expected to be a reference.

**Pull request merged**